cookiecutter.json

```diff
@@ -0,0 +1,4 @@
+{
+    "project_name": "baking-test",
+    "initialize_git": "y"
```

Comment:
Okay, after some testing I realise the entire point of having a dummy initial commit is the essence of providing the initialised git repo, and that the process does actually reuses the users' git config so that the initial commit is in fact done by the current user. 

If anything, I'd like to still reconsider if we still want to support having an initialised git repository, or not. I think this is a decision for us to make. I'm not against it, unless it creates any sort of additional overhead in the template (aside from the actual hook).

Reply:
Resolving, we can leave this subject for now as is, since it's an optional feature and the user can opt-out if needed.